src/adapters/MeetingsJSONAdapter.js

```diff
@@ -342,6 +339,26 @@ export default class MeetingsJSONAdapter extends MeetingsAdapter {
     return stream;
   }
 
+  /**
+   * Returns a promise to a MediaStream object that captures the contents of a screen share.
+   *
+   * @returns {Promise.<MediaStream>} Media stream that captures display
+   * @private
+   */
+  // eslint-disable-next-line class-methods-use-this
+  getShareStream() {
+    const share = document.getElementById('remote-share');
```

Comment:
let's factor this logic into a `getStreamFromElement(id)` function, and then we can call:
```js
meeting.remoteVideo = getStreamFromElement('remote-video');
meeting.remoteShare = getStreamFromElement('remote-share');
```